tests/foreman/cli/test_contentview.py

```diff
@@ -79,122 +77,159 @@
 from robottelo.constants.repos import FAKE_0_INC_UPD_URL
 from robottelo.constants.repos import FAKE_0_PUPPET_REPO
 from robottelo.constants.repos import FAKE_1_YUM_REPO
-from robottelo.constants.repos import FEDORA27_OSTREE_REPO
+from robottelo.constants.repos import OSTREE_REPO
 from robottelo.datafactory import generate_strings_list
-from robottelo.datafactory import invalid_values_list
+from robottelo.datafactory import invalid_names_list
+from robottelo.datafactory import parametrized
+from robottelo.datafactory import valid_names_list
 from robottelo.decorators import skip_if_not_set
 from robottelo.decorators.host import skip_if_os
 from robottelo.helpers import create_repo
 from robottelo.helpers import get_data_file
 from robottelo.helpers import repo_add_updateinfo
-from robottelo.ssh import upload_file
-from robottelo.test import CLITestCase
 from robottelo.vm import VirtualMachine
 from robottelo.vm_capsule import CapsuleVirtualMachine
 
 
-class ContentViewTestCase(CLITestCase):
-    """Content View CLI tests"""
+@pytest.fixture(scope='module')
+def module_rhel_content(module_manifest_org):
```

Comment:
Global as in put it under the `pytest_fixture` file?  If so, then not really since this is only specific to cv.

Global to just this file?  If so, then not really since only the class `Test ContentView` uses it.  In the future, if we add more code coverage that requires this in another class, then sure.

Reply:
A few other components could use a fixture that provides a sync'd repo. I do think it is globally useful :)